new-website/src/3.x/docs/cli-commands/repack-init.md

```diff
@@ -0,0 +1,43 @@
+# repack-init
+
+`@callstack/repack-init` is a command-line tool that helps you to set up Re.Pack in your React Native project. It installs all required dependencies and configures your project to use Re.Pack.
+
+## Usage
+
+```bash
+npx @callstack/repack-init [options]
+```
```

Comment:
would be good to guide the user a bit here, e.g.:
```suggestion
## Usage

In the root of an existing React Native project, run this command in your terminal of choice:

```bash
npx @callstack/repack-init [options]
```
```
feel free to change the wording